octoprint_octorelay/test__init.py

```diff
@@ -10,10 +10,14 @@
 from octoprint.access import ADMIN_GROUP, USER_GROUP
 
 # Patching required before importing OctoRelayPlugin class
-GPIO_mock = Mock()
-GPIO_mock.BCM = "MockedBCM"
-GPIO_mock.OUT = "MockedOUT"
-sys.modules["RPi.GPIO"] = GPIO_mock
+if "RPi.GPIO" in sys.modules:
+    GPIO_mock = sys.modules["RPi.GPIO"]
+else:
+    GPIO_mock = Mock()
+    GPIO_mock.BCM = "MockedBCM"
+    GPIO_mock.OUT = "MockedOUT"
+    sys.modules["RPi.GPIO"] = GPIO_mock
```

Comment:
perhaps we can actually replace GPIO assertions to call-assertions of the driver,
since there are reliable driver tests already.

Reply:
attempted to do this, but it does not look good enough so far